README.md

```diff
@@ -50,3 +50,58 @@ Example:
 ```
 
 [Yarn docs reference for this](https://yarnpkg.com/getting-started/qa#how-to-share-scripts-between-workspaces)
+
+
+## Versioning / Publishing Packages
+
+If you are contributing a user-facing or noteworthy change to a pie-monorepo package that should be added to the changelog, you should include a changeset with your PR.
+
+Changesets are only required for Major, Minor, and Patch changes that have an effect on consumers. Changes that don't affect consumers do not need to be versioned. Examples include linting, testing or CI updates.
+
+To add a changeset, run this script locally in the root of the project:
+
+```bash
+yarn changeset
+```
+
+Follow the prompts to select which package(s) are affected by your change, and whether the change is a major, minor or patch change. This will create a file in the `.changesets` directory of the repo. This change should be committed and included with your PR.
+
+Considerations:
+
+- You can use markdown in your changeset to include code examples, headings, and more. However, **please use plain text for the first line of your changeset**. The formatting of the GitHub release notes does not support headings as the first line of the changeset.
+- When selecting packages for the changesets, only select packages which are intended to be published.
+
+
+### Stable Versions - 'latest' tag
+
+If your change is intended to be released under the `latest` tag on npm, you must follow this workflow:
+
+- Create a branch with your changes. These changes should exclude any `package.json` or manual `CHANGELOG` updates – only include the `.changesets` changes added by Changesets.
+- When you create your PR, target the `main` branch.
+- Upon merging to `main`, a new PR titled **Version Packages** is automatically created. This PR includes the `CHANGELOG.md` and `package.json` version bump. Merging this PR will commit this changes to `main` and execute a publish to npm under the `latest` tag.
+
+
+#### Beta Versions - 'beta' tag
+
+A Beta release is a release that contains experimental changes. These are ready for early adoption and testing by consumers but may introduce bugs (or be considered work-in-progress).
+
+#### Feature versions - 'next' tag
```

Comment:
Looking at this again, I think these to be the same level as stable versions heading (so h3 too?)

```suggestion
### Beta Versions - 'beta' tag

A Beta release is a release that contains experimental changes. These are ready for early adoption and testing by consumers but may introduce bugs (or be considered work-in-progress).

### Feature versions - 'next' tag
```

Reply:
Or you could have a h3 which is called Beta and Feature Release above this title, which might then make sense with the `#### Promoting to stable` title below